__tests__/src/components/WindowTopBar.test.js

```diff
@@ -50,9 +50,9 @@ describe('WindowTopBar', () => {
     expect(wrapper.find(Typography).first().render().text()).toBe('awesome manifest');
   });
 
-  it('passes correct props to <WindowTopBarButtons/>', () => {
+  it('passes correct props to <PluginHook/>', () => {
     const wrapper = createWrapper();
-    expect(wrapper.find(WindowTopBarButtons).first().props().windowId).toBe('xyz');
```

Comment:
Line 9: `import WindowTopBarButtons from '../../../src/containers/WindowTopBarButtons';` can be removed